electron/js/controller.js

```diff
@@ -1,29 +1,22 @@
-var d3 = require('d3');
-var Processor = require('dataproofer').Processing
-var gsheets = require('gsheets')
-var ipc = require("electron").ipcRenderer
-
+var d3 = require("d3");
+var Processor = require("dataproofer").Processing;
+var gsheets = require("gsheets");
+var ipc = require("electron").ipcRenderer;
 
 // libraries required to run test inline
-var requireFromString = require('require-from-string')
+// var requireFromString = require("require-from-string");
```

Comment:
my bad, i think i tried using it and decided against it later. we should delete... oops :)